modules/vba_documents/app/serializers/vba_documents/v1/upload_serializer.rb

```diff
@@ -3,7 +3,7 @@
 module VBADocuments
   module V1
     class UploadSerializer < VBADocuments::UploadSerializer
-      attribute :status
+      set_type :document_upload
```

Comment:
`status` could be removed since it's included in the shared `UploadSerializer` now.

`set_type :document_upload` had to be explicitly included due to [an inheritance problem in the JSONAPI serializer gem](https://github.com/jsonapi-serializer/jsonapi-serializer/issues/113). Without it, `type` was set/defaulted to `upload` based on the name `UploadSerializer`, when it has always been `document_upload`.

Reply:
I was unaware of this bug. Thank you for pointing this out.

Reply:
Looks like the gem we are using [went into maint mode](https://github.com/jsonapi-serializer/jsonapi-serializer) a while ago and the new version [stalled out a year ago](https://github.com/jsonapi-serializer/jsonapi-serializer/pull/141).